build.sbt

```diff
@@ -26,11 +26,7 @@ ThisBuild / nativeBrewInstallCond := Some("matrix.project == 'rootNative'")
 
 lazy val setupCertAndDocker = Seq(
   WorkflowStep.Run(
-    commands = List("chmod 600 world/server.key", "sudo chown 999 world/server.key"),
-    name = Some("Set up cert permissions"),
-  ),
-  WorkflowStep.Run(
-    commands = List("docker-compose up -d"),
+    commands = List("export SERVER_KEY=$(cat world/server.key)", "export SERVER_CERT=$(cat world/server.crt)", "docker-compose up -d"),
```

Comment:
Instead of `export`, should these be set as proper `env = Map(...)` variables for the `WorkflowStep` ?

Reply:
Oh, sorry, I see, they are reading from a file 🤔